packages/astro-ghostcms/src/api/ghostAPI.ts

```diff
@@ -1,23 +1,24 @@
-import { TS_API } from "./content-api";
-import type { Page, Post } from "./content-api/schemas";
+import { TSGhostContentAPI } from "@ts-ghost/content-api";
+import type { Page, Post } from "../schemas/api";
 
 // LOAD ENVIRONMENT VARIABLES
 import { loadEnv } from "vite";
 
-import config from "../integrations/virtual-config";
-
-const CONF_URL = config.ghostURL;
-
 const { CONTENT_API_KEY, CONTENT_API_URL } = loadEnv(
 	"all",
 	process.cwd(),
 	"CONTENT_",
 );
 
-const ghostApiKey = CONTENT_API_KEY;
-const ghostUrl = CONF_URL ? CONF_URL : CONTENT_API_URL;
+// LOAD CONFIG
+import config from "virtual:@matthiesenxyz/astro-ghostcms/config";
```

Comment:
idk if you type this using a `.d.ts` file currently but fyi that's a nice dx improvement

Reply:
Virtual config is just the first step(And yeah, I'm using a `virtual.d.ts` to keep track of types locally XD) , the New Theme Provider will have an entire virtual component system like how `astro-blog` works(Future Plan),  giving more flexibility on top of being able to do full theme takeover like current function